.github/dependabot.yml

```
@@ -5,6 +5,7 @@ updates:
   schedule:
     interval: daily
     time: "10:00"
+  target_branch: "staging"
```

```suggestion
  target-branch: "staging"
```
(as commented in https://github.com/AleoHQ/snarkOS/pull/904)